### DISCUSSION TITLE
Please let us know which chain restaurant you frequent most often.（あなたがよく利用するチェーン経営しているお店を教えてください。）

### DISCUSSION THREAD

**Discussion by ak-naist** [NAIST-SE/AttractingContributors#438] [General]
#### En)Please let us know which chain restaurant you frequent most often.
***
#### Ja)あなたがよく利用するチェーン経営しているお店を教えてください。
***
My fork is here:
https://github.com/ak-naist/AttractingContributors-chain_restaurant_list/tree/aokikatsuki
(奈良には無いけど地元でよく行ってたとかでもOK！)

**Comment by aru1702** [comment]
To be honest, I rare to eat outside
I only several, like Saizeria, McD, KFC, Wendy's, Kappa sushi
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by XC1996114514** [comment]
McD ! Because it is hard to cook the fried chicken

Please help me if you'd like to 🥺🥺🥺
Thanks a lot for your help

https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam
